apps/kyc-server/src/pages/Home.tsx

```
@@ -39,7 +39,7 @@ export default function Home({ message }: HomeProps) {
 			<body>
 				{/* <h1>Kindfi KYC Server HOM</h1> */}
 
-				{/** biome-ignore lint/correctness/useUniqueElementIds: <explanation> */}
+				{/** biome-ignore lint/correctness/useUniqueElementIds: root element */}
```

Biome (2.1.2)

[error] Failed to parse category "lint/correctness/useUniqueElementIds". Example of suppression: // biome-ignore lint: reason

Biome can't recognize the category, usually it's because it doesn't match the expected ones.
Example of suppression: // biome-ignore lint: reason

(suppressions/parse)